File: Examen Parcial/20221309_EX1/Pregunta03/TransitSoft/TransitSoftPersistance/DAO/VehiculoDAO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TransitSoftModel;

namespace TransitSoftPersistance.DAO
{
    public interface VehiculoDAO
    {
        int Insertar(VehiculoDTO vehiculo);

        VehiculoDTO ObtenerPorId(int vehiculoId);

        BindingList<VehiculoDTO> ListarTodos();

        int Modificar(VehiculoDTO vehiculo);

        int Eliminar(VehiculoDTO vehiculo);
    }
}

File: EX2/Conexion RESTful Transitsoft/ServidorRESTFul/TransitSoftPersistance/src/main/java/pe/edu/pucp/transitsoft/dao/VehiculoDAO.java
package pe.edu.pucp.transitsoft.dao;

import java.util.ArrayList;
import pe.edu.pucp.transitsoft.model.VehiculoDTO;

import java.util.ArrayList;

public interface VehiculoDAO {
    
    public Integer insertar(VehiculoDTO vehiculo);
    
    public VehiculoDTO obtenerPorId(Integer id);
    
    public ArrayList<VehiculoDTO> listarTodos();
    
    public Integer modificar(VehiculoDTO vehiculo);
    
    public Integer eliminar(VehiculoDTO vehiculo);
}